tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_forget_wifi.py

```diff
@@ -0,0 +1,25 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.settings.app import Settings
+
+
+class TestForgetWifi(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.data_layer.disable_wifi()
+
+    def test_connect_to_wifi_and_forget_active_wifi(self):
+        """
+        https://moztrap.mozilla.org/manage/case/15898/
+        """
+
+        settings = Settings(self.marionette)
+        settings.launch()
+
+        wifi_settings = settings.open_wifi()
+
+        wifi_settings.enable_wifi()
+        wifi_settings.connect_to_network(self.testvars['wifi'])
+
```

Comment:
I would like to see an assert like in test_settings_wifi.py here:
self.assertTrue(self.data_layer.is_wifi_connected(self.testvars['wifi']), "WiFi was not connected via Settings app")